py/surveyqa/nightly.py

```diff
@@ -126,6 +126,27 @@ def plot_timeseries(source, name, color, tools=None, x_range=None, title=None, t
 
     return fig
 
+def hourangle_timeseries(width=600, height=200, min_border_left=50, min_border_right=50):
+    '''Placeholder for a hour angle vs. time plot'''
+    p = bk.figure(plot_width=width, plot_height=height,
+                  x_axis_label='Time',
+                  y_axis_label='Hour Angle', min_border_left=min_border_left,
+                  min_border_right=min_border_right)
+    p.xaxis.axis_label_text_color='#ffffff'
+    p.toolbar_location = None
+    return p
+
+def brightness_timeseries(width=600, height=200, min_border_left=50, min_border_right=50):
+    '''Placeholder for a sky brightness timeseries plot'''
+    p = bk.figure(plot_width=width, plot_height=height,
+                  x_axis_label='Time',
+                  y_axis_label='Sky Brightness',
+                  min_border_left=min_border_left,
+                  min_border_right=min_border_right)
+    p.xaxis.axis_label_text_color='#ffffff'
+    p.toolbar_location = None
+    return p
+
```

Comment:
I think these two were removed by an earlier PR and are accidentally being added back in.  Double check that they aren't used anymore and remove from this PR if possible.